lib/commands/config/config-set.ts

```diff
@@ -1,12 +1,19 @@
 export class ConfigApplyCommand implements ICommand {
-	constructor(private $nsCloudServerConfigManager: IServerConfigManager,
-		private $stringParameterBuilder: IStringParameterBuilder) { }
-
 	public allowedParameters: ICommandParameter[] = [this.$stringParameterBuilder.createMandatoryParameter("You must specify path to configuration file.")];
 
+	public get dashedOptions() {
+		return this.$nsCloudOptionsProvider.dashedOptions;
+	}
+
+	constructor(private $nsCloudOptionsProvider: ICloudOptionsProvider,
+		private $nsCloudServerConfigManager: IServerConfigManager,
+		private $options: ICloudOptions,
+		private $stringParameterBuilder: IStringParameterBuilder) { }
+
 	public async execute(args: string[]): Promise<void> {
 		const configurationFile = args[0];
-		this.$nsCloudServerConfigManager.applyConfig(null, { configurationFile: configurationFile });
+		const globalServerConfig = { apiVersion: this.$options.apiVersion, serverProto: this.$options.serverProto };
```

Comment:
Should you decide to follow my suggestion from the previous comment it can be applied here as well